src/test/java/com/github/mongobeej/utils/ProfileMatcherTest.java

```diff
@@ -0,0 +1,98 @@
+package com.github.mongobeej.utils;
+
+import org.junit.Test;
+import org.springframework.context.annotation.Profile;
+
+import java.lang.reflect.AnnotatedElement;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ProfileMatcherTest {
+
+  private ProfileMatcher profileMatcher;
+
+  @Test
+  public void matchesPositive() throws Exception {
+    profileMatcher = new ProfileMatcher(asList("local"));
```

Comment:
minor: actually if you set different value to `profileMatcher` then make it a local variable, not a class field